src/public/modules/core/componentViews/footer.html

```diff
@@ -0,0 +1,117 @@
+<footer class="landing__footer">
+  <div class="footer__section footer__section--bottom-border">
+    <div class="footer__tagline">
+      <h1 class="tagline--name" translate="FOOTER.APP_NAME"></h1>
+      <span class="tagline--caption" translate="FOOTER.APP_TEXT"></span>
+    </div>
+    <div class="footer__links">
+      <a
+        class="footer__links--link"
+        translate-attr="{ href: 'FOOTER.USER_GUIDE.HREF' }"
+        translate="FOOTER.USER_GUIDE.TEXT"
+        target="_blank"
+      ></a>
+      <a
+        class="footer__links--link"
+        ui-sref="privacy"
+        target="_blank"
+        translate="FOOTER.PRIVACY.TEXT"
+      ></a>
+      <a
+        class="footer__links--link"
+        ui-sref="terms"
+        target="_blank"
+        translate="FOOTER.TOS.TEXT"
+      ></a>
+      <a
+        class="footer__links--link footer__links--link--last"
```

Comment:
for the record I don't think this is what BEM is supposed to look like lol but probably not worth fixing

Reply:
no h8, no h8